src/types/validate.go

```diff
@@ -40,27 +40,28 @@ func SupportedOS() []string {
 
 // Validate runs all validation checks on the package.
 func (pkg ZarfPackage) Validate() error {
+	errs := []error{}
 	if pkg.Kind == ZarfInitConfig && pkg.Metadata.YOLO {
-		return fmt.Errorf(lang.PkgValidateErrInitNoYOLO)
+		errs = append(errs, fmt.Errorf(lang.PkgValidateErrInitNoYOLO))
```

Comment:
Could we use `errors.Join` as an append instead of making a `errs := []error{}`
I would say either do a `var err error` and then `errors.Join(error, fmtErrorf(...)`